src/components/FilterBarHub.tsx

```diff
@@ -6,44 +6,54 @@ import { DropdownFilterBarItem } from "azure-devops-ui/Dropdown";
 import {
   TeamProjectReference,
   ProjectInfo,
-  WebApiTagDefinition
+  WebApiTagDefinition,
 } from "azure-devops-extension-api/Core/Core";
 import { Filter } from "azure-devops-ui/Utilities/Filter";
 import { IListBoxItem } from "azure-devops-ui/ListBox";
 import {
   DropdownSelection,
-  DropdownMultiSelection
+  DropdownMultiSelection,
 } from "azure-devops-ui/Utilities/DropdownSelection";
 import {
   GitRepository,
-  IdentityRefWithVote
+  IdentityRefWithVote,
 } from "azure-devops-extension-api/Git/Git";
 import * as Data from "../tabs/PulRequestsTabData";
 import { IdentityRef } from "azure-devops-extension-api/WebApi/WebApi";
 import { getVoteDescription } from "./Columns";
 import { ITableColumn } from "azure-devops-ui/Table";
 import { Status } from "azure-devops-ui/Status";
 import { getStatusSizeValue, getStatusIcon } from "../models/constants";
+import { PullRequestModel } from "../models/PullRequestModel";
+import { Spinner } from "office-ui-fabric-react";
 
-export const myApprovalStatuses = Object.keys(Data.ReviewerVoteOption)
-  .filter(value => !isNaN(parseInt(value, 10)))
```

Comment:
Why don't you rename this variable into something more meaningful?

Reply:
Do you mean the myApprovalStatuses?

Reply:
I mean `value` in the `filter` function.